deploy/overlays/svobs/kustomization.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Kustomization for svobs website running on GCP `datcom-website-statvar-migrate` project.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+nameSuffix: -svobs
+namespace: website
+
+resources:
+  - ../../svobs
+
+configMapGenerator:
+  - name: website-configmap
+    behavior: create
+    literals:
+      - flaskEnv=autopush
+      - gcsBucket=datcom-website-statvar-migrate-resources
+      - secretProject=datcom-website-statvar-migrate
+  - name: mixer-configmap
+    behavior: create
+    literals:
+      - mixerProject=datcom-website-statvar-migrate
+      - serviceName=website-esp.endpoints.datcom-website-statvar-migrate.cloud.goog
+
+patchesStrategicMerge:
+  - |-
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: website-app
+    spec:
+      replicas: 6
+
+patchesJson6902:
```

Comment:
could you describe what is this patch for? i'm curious where "json6902" came from